airflow/update-environment-variables.sh

```diff
@@ -1,31 +1,40 @@
 #!/usr/bin/env bash
 echo Input ingestion service url?
-read ingestionUrl
+INGESTION_URL=$(gcloud run services list --platform managed --filter ingestion-service | grep -o "https:.*\.app")
```

Comment:
You can use --format 'value(status.url)' instead of grepping.

I also prefer using gcloud run services describe instead of list since it is more direct, but I'll leave that up to you since it requires a region and I'm not sure how straightforward it is to get that here (I think we could hard code it but that might be annoying later)

Reply:
I switched to using the format value :)  Ima leave the rest as is as its working and I dont wanna spend too much time on it.